config/vufind/Demo.ini

```diff
@@ -236,6 +236,16 @@ extraHoldFields = "comments:requestGroup:pickUpLocation:proxiedUsers:startDate:r
 ; Should we include a link to a fake OPAC?
 fakeOpacLink = false
 
+; Block ILS methods during given times. Format is "method:setting = value"
+; method   - Method in ILS driver
+; settting - Valid settings are startDate, endDate, recurringStart, recurringEnd
+; value    - Date for startDate/endDate, time for recurringStart/recurringEnd
```

Comment:
Maybe worth commenting on preferred date/time formats here (whether that's as simple as saying "compatible with strtotime" or a little more complex is up to you).

Reply:
I went with instructing to use YYYY-MM-DD and HH:MM. Is that enough? `strtotime` will catch other formats as well, but as you pointed out in the other comment, something like "-2 days" might cause a problem.

Reply:
Yes, that makes sense to me!